### PR TITLE
Thu/upgrade node version to v16

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

**Review by ardvi — APPROVED** [review]
LGTM :+1:

File: README.md [hunk 1]
```diff
@@ -54,16 +54,50 @@ Please refer to [Webapp configuration](doc/README.md)
...
+This project is configured to be used with the [Yarn](https://yarnpkg.com/getting-started/install) package manager. You
+can use NPM (the node package manager) to install yarn globally with:
+
+```bash
+npm install --global yarn
+```
+
+### Local azure function
+
+This section describes how to run a local Azure Function, in order to provide the webapp users with a valid token for the PowerBI service. **This step is only necessary if you use the "service account" mode to share PowerBI charts**. If you prefer to let users communicate with PowerBI with their own accounts, then the Azure Function won't be necessary, and you can skip this section (please refer to [doc/powerBI.md](/home/thuet/dev/azure-sample-webapp/doc/powerBI.md) for the charts configuration documentation).
```
Comment (esasova) [inline README.md:86]: I think it's not a good link to PowerBI doc :slightly_smiling_face:

File: README.md [hunk 2]
```diff
@@ -54,16 +54,50 @@ Please refer to [Webapp configuration](doc/README.md)
...
+You can then start the Azure Functions, that are required for the PowerBI embedded reports to work correctly, by running
+the commands below:
 
 ```
 cd api
 yarn install
 yarn start
 ```
 
+### Local web-app server
```
Comment (esasova) [inline README.md:99]: in this doc we always spell webapp in one word, but here it is spelled with a hyphen :thinking:

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: README.md (hunk 2 above)
Comment (csm-thu) [reply README.md]: Fixed :slightly_smiling_face:

**Pull request merged**